sw/device/lib/crypto/impl/keyblob_unittest.cc

```diff
@@ -8,14 +8,15 @@
 
 #include "gmock/gmock.h"
 #include "gtest/gtest.h"
-#include "sw/device/lib/base/hardened_status.h"
 #include "sw/device/lib/crypto/impl/status.h"
 #include "sw/device/lib/crypto/include/datatypes.h"
 
 namespace keyblob_unittest {
 namespace {
 using ::testing::ElementsAreArray;
 
+#define EXPECT_OK(status_) EXPECT_EQ(status_.value, OTCRYPTO_OK.value)
```

Comment:
Consider moving this macro to a testutils library if planning to use in other unittest modules.

Reply:
Yep, I'll move it if I ever use it in another unit test. But I think long-term there might be a more elegant way to do this -- it was just the smallest diff to get the keyblob tests to compile.